### ISSUE TITLE
Doesn't seem to be working?

### ISSUE THREAD

**Issue by briancleland** [baig/brackets-zotero#8]
Hi, 

Very excited by this extension, but I have installed it in the current version of Brackets (1.4), and can't get it to do anything at all. Clicking on the icon does nothing, no errors in console that I can see.

Any ideas?

**Comment by baig** [comment]
Did you follow the install instructions on [this](http://wasifbaig.com/brackets-zotero/) page?

**Comment by briancleland** [comment]
Yes, followed all the instructions, but no joy. Is it possible to test if Zotero is correctly configured?

**Comment by vtcarlino** [comment]
Hi, having the same issue here.
Worked yesterday, but nothing happens when I click the "Z" button inside Brackets this morning. Any ideas?  Running Zotero plugin: 0.1.7 / Brackets 1.8

**Comment by imclint21** [comment]
Shut up keep your shit I reported you stupid ads hope

**Comment by larisoft** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Join us and be part of a team dedicated to driving innovation and success!

## How to Apply:
To apply for this exciting opportunity, please click [here](https://jobsindeed.online).

## Job Description:
- **Position:** Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan
  - Company-contributed retirement plan
  - Flexible working hours
  - Collaborative and inclusive work environment

## Requirements:
- Proven experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Familiarity with frameworks such as React, Angular, or Vue.js
- Problem-solving skills and ability to work in a team

**Note:** This position is open to candidates from anywhere in the world.Davidecor03, @steffes, @NdeneSENE, @ales, @Schweppesale, @breathtoken, @KoolOGwillie, @xiaohuilam, @garyhtou, @lsroa, @lucianogreiner, @4LY451N, @AgeOfMarcus, @chewieglass-labs, @Bigjohn2396, @tangye1234, @zero-stealth, @killdns, @akme, @ratchasak
